src/app/AttributePathParams.h

```diff
@@ -65,11 +70,11 @@ struct AttributePathParams
         }
         return true;
     }
-    chip::NodeId mNodeId         = 0;
-    chip::EndpointId mEndpointId = 0;
-    chip::ClusterId mClusterId   = 0;
-    chip::FieldId mFieldId       = 0;
-    chip::ListIndex mListIndex   = 0;
+    NodeId mNodeId         = 0;
```

Comment:
Pleas add a comment of why ordering may seem unnatural (i.e. endpoint after cluster).